Translations/pl.lproj/InfoPlist.strings

```diff
@@ -2,21 +2,21 @@
 "7Z Archive" = "archiwum 7Z"
 "LZMA Archive" = "archiwum LZMA"
 "LZOP Archive" = "archiwum LZOP"
-"LZOP Tarball Archive" = "skompresowane archiwum TAR LZOP"
+"LZOP Tarball Archive" = "archiwum TAR LZOP"
 "LZIP Archive" = "archiwum LZIP"
-"LZIP Tarball Archive" = "skompresowane archiwum TAR LZIP"
+"LZIP Tarball Archive" = "archiwum TAR LZIP"
 "ZIP Archive" = "archiwum ZIP"
 "XZ Archive" = "archiwum XZ"
-"XZ Tarball Archive" = "skompresowane archiwum TAR XZ"
+"XZ Tarball Archive" = "archiwum TAR XZ"
 "RAR Archive" = "archiwum RAR"
 "RAR Legacy Archive Volume" = "RAR Legacy Archive Volume"
```

Comment:
Can you also translate that and the next line? 😀

Reply:
If you mean RAR Legacy Archive Volume and Generic Archive Volume - I thought about this for a while and finally left those untranslated on purpose. In my opinion, there is no way/need to translate it to Polish, since those are file extension names... 😄